1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
+
+Si has respondido que sí, ¿cómo lo haces?
+- Acudo directamente a webs de venta de entradas 
+- Escribo en mi navegador frases tipo: “entradas para concierto X”  
+- Acudo a la página web del artista/grupo  
+- Acudo a páginas web musicales que me informan sobre la venta de entradas 
+- Ninguna de las anteriores
+
+Una vez realizada la encuesta y analizadas las respuestas obtenidas, los resultados y las conclusiones que hemos sacado son las siguientes:
+
+**Tipo de formato y contenido para consumir información musical**
+
+Las primeras dos preguntas de la encuesta han sido sobre el tipo de formato y contenido que utilizan a la hora de consumir información sobre música. En el caso del tipo de formato, los resultados han sido muy parecidos. El porcentaje más alto (80% de los encuestados) y por tanto, el tipo de contenido que más consumen son el texto, el vídeo y la imagen. El audio ha obtenido un porcentaje bastante más bajo (20%), pero al obtener un porcentaje que no sea el 0, creemos que también podría ser una forma de insertar contenido en nuestra página web, aunque no se utilizará más que los demás formatos.
+
+En cuanto a los resultados del tipo de contenido que consumen en Internet, los porcentajes han sido más variados. Las entrevistas y los audios son los formatos que más se consumen por parte de los encuestados, ya que todas las personas que han rellenado la encuesta las han elegido. Por detrás de las entrevistas y los audios estarían las noticias con un porcentaje del 80%. Los videoclips y los reportajes también han sido elegidos pero con un porcentaje bastante menor. En última posición están las críticas con un 0% de porcentaje, por tanto, ninguno de los encuestados consume este tipo de contenido. Aun así, creemos que es conveniente optar por diferentes tipos de contenidos y por esta razón, no descartaremos ninguno de ellos, por el momento.
+
+**Interés sobre los géneros musicales**
+
+Parece ser que el género que genera más interés es el pop, seguido del reggaetón, el rock y el trap. Estos datos nos indican que aunque generemos contenido de diferentes géneros musicales, es conveniente ofrecer más espacio al pop, sobre todo, pero también al reggaetón, el rock y el trap.
+
+Por otro lado, se ha observado que los géneros que crean menos interés con una media de un 1 son el flamenco y el country, por lo que puede que se descarte ofrecer información sobre estos estilos musicales. En cuanto al resto de los géneros, es decir, el jazz, punk, rap, techno y la música clásica, han obtenido una media de 3 puntos sobre 5. Esto quiere decir que aunque estos géneros musicales no sean los preferidos de nuestro público objetivo, tiene un interés en obtener información acerca de ellos. También se les ha ofrecido a los encuestados la posibilidad de indicar algún otro género musical, pero no ha habido respuesta alguna. Por esta razón, descartamos ofrecer información acerca de cualquier género que no se haya nombrado.
```

Comment:
Aquí tomáis la decisión de no descartar ningún genero que haya obtenido más de una mención en vuestras encuestas, pero no sé cuál es la distancia con los más populares. Ha surgido esta cuestión antes, pero deberías pensar muy bien (y esto afectará a vuestra visión) si vais a ser un medio musical generalista (todos o casi todos los géneros) o vais a centraros en los más populares (a día de hoy pop, reggaeton, rock y trap).